graphql/schema/directives.graphql

```diff
@@ -1,2 +1,3 @@
 directive @requireSuperUser on FIELD_DEFINITION 
-directive @requireProjectAccess(access: ProjectSettingsAccess!) on ARGUMENT_DEFINITION | INPUT_FIELD_DEFINITION
\ No newline at end of file
+directive @requireProjectAccess(access: ProjectSettingsAccess!) on ARGUMENT_DEFINITION | INPUT_FIELD_DEFINITION | FIELD_DEFINITION
+directive @restrictProjectAccess on FIELD_DEFINITION
```

Comment:
nit. It would be awesome if you added some comments to these directives and how they should be used so we can get intellisense support. 
<img width="800" alt="image" src="https://user-images.githubusercontent.com/4605522/217387462-5c84b03d-0a60-453f-849d-c38c20f5fc42.png">